pkg/kev/kev_reconcile_test.go

```diff
@@ -491,6 +491,18 @@ var _ = Describe("Reconcile", func() {
 						HaveKeyWithValue(config.LabelWorkloadLivenessProbeHTTPPath, "/status"))
 					Expect(svcCfg.GetLabels()).NotTo(HaveKey(config.LabelWorkloadLivenessProbeCommand))
 				})
+
+				It("should have a valid http readiness probe", func() {
+					svcCfg, err := env.GetService("wordpress")
+					Expect(err).To(Succeed())
+					Expect(svcCfg.GetLabels()).To(
+						HaveKeyWithValue(config.LabelWorkloadReadinessProbeType, kubernetes.ProbeTypeHTTP.String()))
+					Expect(svcCfg.GetLabels()).To(
+						HaveKeyWithValue(config.LabelWorkloadReadinessProbeHTTPPort, "8080"))
+					Expect(svcCfg.GetLabels()).To(
+						HaveKeyWithValue(config.LabelWorkloadReadinessProbeHTTPPath, "/status"))
+					Expect(svcCfg.GetLabels()).NotTo(HaveKey(config.LabelWorkloadLivenessProbeCommand))
```

Comment:
did you mean `config.LabelWorkloadReadinessProbeCommand`?

Reply:
copy paste leftover, good catch 🙏